string object_id
string class_name
geometry_msgs/Pose pose
geometry_msgs/Vector3 dimensions
float64 confidence
